app/constants/index.ts

```diff
@@ -24,8 +25,10 @@ export const STATUS_PAGE_URL = 'http://status.test-blockstack.com';
 
 export const DEFAULT_STACKS_NODE_URL = 'https://stacks-node-api.blockstack.org';
 
-export const EXPLORER_URL =
-  NETWORK === 'testnet' ? 'https://testnet-explorer.blockstack.org' : 'https://explorer.stacks.co';
+export const EXPLORER_URL = whenNetwork<string>({
+  mainnet: 'https://explorer.stacks.co',
```

Comment:
A lot of these string paths i wish were consts personally

Reply:
this is in the constants file?